app/workers/restore_apicast_master_token_worker.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# This will only perform in the context of k8s or OpenShift
+# It sets the value of the access token from an environment variable
+class RestoreApicastMasterTokenWorker < ApplicationJob
+  unique :until_executed,
+         on_conflict: ->(job) { Rails.logger.info "Another job is already scheduled for: #{job.class} #{job.arguments}" }
```

Comment:
It looks like we'll be reusing this proc in the future. Worth defining it (as a class helper method) in `ApplicationJob`, or in a mixin?

I know it can get ugly, so feel free to ignore if you prefer starting as-is for now.

Reply:
I think it is a fair point.

Reply:
Yes why not :)

Also I decided to use this gem as it is ActiveJob compatible, the sidekiq-lock one is not maintained since 3 years (probably because it is stable enough but who knows it is really scary)